Helper/Cache/CacheHelper.php

```diff
@@ -53,4 +55,20 @@ public function save(CacheItem $item, array $data)
 
         $this->cache->save($item);
     }
+
+    public static function setCacheHeaders(Request $request, Response $response): Response
```

Comment:
why are we returning the $response? isn't it passed as an object / modified by reference?
```suggestion
    public static function setCacheHeaders(Request $request, Response $response): void
```

Reply:
And the name is "strange" I would expect a "setter" to be setting something in the class that we are currently in. I propose c
```suggestion
    public static function makeResponseCacheable(Request $request, Response $response): void
```

Reply:
True I aggree